snap/info.go

```diff
@@ -253,8 +253,8 @@ type ChannelSnapInfo struct {
 	Size        int64           `json:"size"`
 }
 
-// Name returns the blessed name for the snap.
-func (s *Info) Name() string {
+// InstanceName returns the blessed name for the snap.
```

Comment:
The documentation is invalid IMO.

`InstanceName returns the blessed name of the snap decorated with instance key, if any`